hpe3parclient/client.py

```diff
@@ -4353,7 +4353,7 @@ def rcopyLinkExists(self, targetName, local_port, target_system_peer_port):
         return rcopylink_exits
 
     def admitRemoteCopyTarget(self, targetName, mode, remote_copy_group_name,
-                              source_target_volume_pairs_list=[]):
+                              source_target_volumes_dict=None):
```

Comment:
Please change this to optional=None as there could be other arguments added as optional parameters in future. Populate optional with other parameters including this one

Also, please change to a list:
Eg,
[{'source_volume': <source_volume_name>, 'target_volume': <target_volume_name>},...]